postgresmodels/containerscan.go

```diff
@@ -41,7 +41,7 @@ type VulnerabilityScanSummary struct {
 	BaseModel
 	ScanKind                   string
 	ImageScanId                string `gorm:"primaryKey"`
-	ContainerSpecId            string
+	ContainerSpecId            string // ContainerSpecId = ContainersScanID
```

Comment:
This is not true. The ContainerSpecId is the container spec ID. The in-cluster sends it inside the "scanID" fieild. But this is not scan ID,